src/resolver.ts

```diff
@@ -59,32 +74,15 @@ interface JsonWebKey {
   [x: string]: any
 }
 
-export interface PublicKey {
+export interface VerificationMethod {
```

Comment:
The DID spec registries still list `publicKey`, even if deprecated. I think it should be kept and marked with
```typescript
/**
 * @deprecated
 */
```

I have the same comment for the `publicKey` property of `DIDDocument`

Reply:
K, fair enough. Will add it back to the `DIDDocument`.